benchmarks/benchmarks/bench_core.py

```diff
@@ -215,13 +215,39 @@ class Indices(Benchmark):
     def time_indices(self):
         np.indices((1000, 500))
 
-class VarComplex(Benchmark):
-    params = [10**n for n in range(0, 9)]
-    def setup(self, n):
-        self.arr = np.random.randn(n) + 1j * np.random.randn(n)
-
-    def teardown(self, n):
-        del self.arr
-
-    def time_var(self, n):
-        self.arr.var()
+class StdStatsMethods(Benchmark):
+    params = [['int8', 'uint8', 'int16', 'uint16', 'int32', 'uint32',
+              'int64', 'uint64', 'float32', 'float64', 'intp',
+              'complex64', 'complex128', 'complex256',
+                'bool', 'float', 'int', 'complex'],
+              [1000**n for n in range(0, 2)]]
+    param_names = ['dtype', 'size']
+
+    def setup(self, dtype, size):
+        try:
+            self.data = np.ones(size, dtype=getattr(np, dtype))
+        except AttributeError:  # builtins throw AttributeError after 1.20
+            self.data = np.ones(size, dtype=dtype)
```

Comment:
I don't really see the point of the try part here, but OK.

Reply:
Can't seem to reproduce where I ran into that now, but probably best to leave it in (haven't run the full set, takes a few days).